app/routes/settings+/profile.tsx

```diff
@@ -1,12 +1,12 @@
 import { json, type DataFunctionArgs } from '@remix-run/node'
 import { Link, Outlet, useMatches } from '@remix-run/react'
+import { z } from 'zod'
```

Comment:
This was a eslint issue I saw in github

![Screenshot 2023-09-18 at 9 07 36 AM](https://github.com/epicweb-dev/epic-stack/assets/5374985/114dead0-34b8-4783-aea8-0f86f4fb20bf)